app/jobs/zendesk_tickets_job.rb

```diff
@@ -0,0 +1,23 @@
+class ZendeskTicketsJob < ActiveJob::Base
+  queue_as :zendesk
+
+  def perform(feedback)
+    ZendeskAPI::Ticket.create!(
+      Rails.configuration.zendesk_client,
+      description: feedback.body,
+      requester: {
+        email: feedback.email_address,
+        name: 'Unknown'
+      },
+      custom_fields: custom_fields(feedback)
+    )
+  end
+
+  def custom_fields(feedback)
+    [
+      { id: '23730083', value: feedback.referrer },
+      { id: '23757677', value: 'prison_visits' },
+      { id: '23791776', value: feedback.user_agent }
```

Comment:
I'm not sure! They're like this in the old app, but there's no explanation (except our old friend 'Initial commit (open sourcing)').